--- Shared/Localizations/en.lproj/Localizable.strings
+++ Shared/Localizations/en.lproj/Localizable.strings
@@ -147,7 +147,7 @@
 // Application Section
 "APPS_INFORMATION_SECTION_TITLE_APPLICATION" = "Application";
 // Bundle Section
-"APPS_INFORMATION_SECTION_TITLE_NAME" = "Bundle";
+"APPS_INFORMATION_SECTION_TITLE_BUNDLE" = "Bundle";
 
 //
 "APPS_INFORMATION_TITLE_DELETED_FILE" = "Deleted File";
